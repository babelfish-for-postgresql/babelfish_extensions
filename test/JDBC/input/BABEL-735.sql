select exp()
go

select exp(null)
go

select exp(-4321)
go

select exp(0)
go

select exp(4321)
go

select exp(3214.123)
go

select exp(-12.23)
go

select exp(123143234534523434545645456456)
go

select exp(-1233435465465567645645645645)
go

select exp('23122312.2342342342')
go

select exp('-132973942.239874234')
go

select exp('213.123')
go

select exp('1233.1231')
go

select exp('-1293797.21973813')
go

select exp(cast(12 as int))
go

select exp(cast(-1234.1234 as int))
go

select exp(cast(-12.15 as int))
go

select exp(cast(12.15 as int))
go

select exp(cast(-1234.1234 as smallint))
go

select exp(cast(-12.15 as smallint))
go

select exp(cast(12.15 as smallint))
go

select exp(cast(12 as tinyint))
go

select exp(cast(12 as bigint))
go

select exp(cast(-1234.1234 as bigint))
go

select exp(cast(-12.15 as bigint))
go

select exp(cast(12.15 as bigint))
go

select exp(cast(-1234.1234 as money))
go

select exp(cast(-12.15 as money))
go

select exp(cast(12.15 as money))
go

select exp(cast(-1234.1234 as smallmoney))
go

select exp(cast(-12.15 as smallmoney))
go

select exp(cast(12.15 as smallmoney))
go

select exp(cast(-1234.1234 as numeric))
go

select exp(cast(-12.15 as numeric))
go

select exp(cast(12.15 as numeric))
go

select exp(cast(-1234.1234 as decimal))
go

select exp(cast(-12.15 as decimal))
go

select exp(cast(12.15 as decimal))
go

select exp(cast(-1234.1234 as float))
go

select exp(cast(-12.15 as float))
go

select exp(cast(12.15 as float))
go

select exp(cast(12 as real))
go

select exp(cast(-1234.1234 as real))
go

select exp(cast(-12.15 as real))
go

select exp(cast(12.15 as real))
go

select exp(cast(12 as binary))
go

select exp(cast(-1234 as binary))
go

select exp(cast(-12 as binary))
go

select exp(cast(12 as varbinary))
go

select exp(cast(-12 as varbinary))
go

select exp(cast('-12.12' as varchar))
go

select exp(cast('1235.1234' as varchar))
go

select exp(cast('12.16' as varchar))
go

select exp(cast('-12.12' as nvarchar))
go

select exp(cast('1235.1234' as nvarchar))
go

select exp(cast('12.16' as nvarchar))
go

select exp(cast('-12.12' as text))
go

select exp(cast('1235.1234' as text))
go

select exp(cast('12.16' as text))
go

select exp(cast('-12.12' as ntext))
go

select exp(cast('1235.1234' as ntext))
go

select exp(cast('12.16' as ntext))
go
