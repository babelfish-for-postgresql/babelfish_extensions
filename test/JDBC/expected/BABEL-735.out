select exp()
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: function exp() does not exist)~~


select exp(null)
go
~~START~~
float
<NULL>
~~END~~


select exp(-4321)
go
~~START~~
float
0.0
~~END~~


select exp(0)
go
~~START~~
float
1.0
~~END~~


select exp(4321)
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: value out of range: overflow)~~


select exp(3214.123)
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: value out of range: overflow)~~


select exp(-12.23)
go
~~START~~
float
4.8817831756353485E-6
~~END~~


select exp(123143234534523434545645456456)
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: value out of range: overflow)~~


select exp(-1233435465465567645645645645)
go
~~START~~
float
0.0
~~END~~


select exp('23122312.2342342342')
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: value out of range: overflow)~~


select exp('-132973942.239874234')
go
~~START~~
float
0.0
~~END~~


select exp('213.123')
go
~~START~~
float
3.6152877254571996E92
~~END~~


select exp('1233.1231')
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: value out of range: overflow)~~


select exp('-1293797.21973813')
go
~~START~~
float
0.0
~~END~~


select exp(cast(12 as int))
go
~~START~~
float
162754.79141900392
~~END~~


select exp(cast(-1234.1234 as int))
go
~~START~~
float
0.0
~~END~~


select exp(cast(-12.15 as int))
go
~~START~~
float
6.14421235332821E-6
~~END~~


select exp(cast(12.15 as int))
go
~~START~~
float
162754.79141900392
~~END~~


select exp(cast(-1234.1234 as smallint))
go
~~START~~
float
0.0
~~END~~


select exp(cast(-12.15 as smallint))
go
~~START~~
float
6.14421235332821E-6
~~END~~


select exp(cast(12.15 as smallint))
go
~~START~~
float
162754.79141900392
~~END~~


select exp(cast(12 as tinyint))
go
~~START~~
float
162754.79141900392
~~END~~


select exp(cast(12 as bigint))
go
~~START~~
float
162754.79141900392
~~END~~


select exp(cast(-1234.1234 as bigint))
go
~~START~~
float
0.0
~~END~~


select exp(cast(-12.15 as bigint))
go
~~START~~
float
6.14421235332821E-6
~~END~~


select exp(cast(12.15 as bigint))
go
~~START~~
float
162754.79141900392
~~END~~


select exp(cast(-1234.1234 as money))
go
~~START~~
float
0.0
~~END~~


select exp(cast(-12.15 as money))
go
~~START~~
float
5.288372581358964E-6
~~END~~


select exp(cast(12.15 as money))
go
~~START~~
float
189094.08983869816
~~END~~


select exp(cast(-1234.1234 as smallmoney))
go
~~START~~
float
0.0
~~END~~


select exp(cast(-12.15 as smallmoney))
go
~~START~~
float
5.288372581358964E-6
~~END~~


select exp(cast(12.15 as smallmoney))
go
~~START~~
float
189094.08983869816
~~END~~


select exp(cast(-1234.1234 as numeric))
go
~~START~~
float
0.0
~~END~~


select exp(cast(-12.15 as numeric))
go
~~START~~
float
6.14421235332821E-6
~~END~~


select exp(cast(12.15 as numeric))
go
~~START~~
float
162754.79141900392
~~END~~


select exp(cast(-1234.1234 as decimal))
go
~~START~~
float
0.0
~~END~~


select exp(cast(-12.15 as decimal))
go
~~START~~
float
6.14421235332821E-6
~~END~~


select exp(cast(12.15 as decimal))
go
~~START~~
float
162754.79141900392
~~END~~


select exp(cast(-1234.1234 as float))
go
~~START~~
float
0.0
~~END~~


select exp(cast(-12.15 as float))
go
~~START~~
float
5.288372581358964E-6
~~END~~


select exp(cast(12.15 as float))
go
~~START~~
float
189094.08983869816
~~END~~


select exp(cast(12 as real))
go
~~START~~
float
162754.79141900392
~~END~~


select exp(cast(-1234.1234 as real))
go
~~START~~
float
0.0
~~END~~


select exp(cast(-12.15 as real))
go
~~START~~
float
5.288374598713393E-6
~~END~~


select exp(cast(12.15 as real))
go
~~START~~
float
189094.0177050411
~~END~~


select exp(cast(12 as binary))
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: function exp("binary") does not exist)~~


select exp(cast(-1234 as binary))
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: function exp("binary") does not exist)~~


select exp(cast(-12 as binary))
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: function exp("binary") does not exist)~~


select exp(cast(12 as varbinary))
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: function exp(varbinary) does not exist)~~


select exp(cast(-12 as varbinary))
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: function exp(varbinary) does not exist)~~


select exp(cast('-12.12' as varchar))
go
~~START~~
float
5.449427503696814E-6
~~END~~


select exp(cast('1235.1234' as varchar))
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: value out of range: overflow)~~


select exp(cast('12.16' as varchar))
go
~~START~~
float
190994.5170362057
~~END~~


select exp(cast('-12.12' as nvarchar))
go
~~START~~
float
5.449427503696814E-6
~~END~~


select exp(cast('1235.1234' as nvarchar))
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: value out of range: overflow)~~


select exp(cast('12.16' as nvarchar))
go
~~START~~
float
190994.5170362057
~~END~~


select exp(cast('-12.12' as text))
go
~~START~~
float
5.449427503696814E-6
~~END~~


select exp(cast('1235.1234' as text))
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: value out of range: overflow)~~


select exp(cast('12.16' as text))
go
~~START~~
float
190994.5170362057
~~END~~


select exp(cast('-12.12' as ntext))
go
~~START~~
float
5.449427503696814E-6
~~END~~


select exp(cast('1235.1234' as ntext))
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: value out of range: overflow)~~


select exp(cast('12.16' as ntext))
go
~~START~~
float
190994.5170362057
~~END~~

